### ISSUE TITLE
No reaction on comments

### ISSUE THREAD

**Issue by Woitekku** [trstringer/manual-approval#70]
Hey,

I'm using it with following:

```       - name: Approval needed
        uses: trstringer/manual-approval@v1
        with:
          secret: ...
          approvers: ...
          minimum-approvals: ...
          issue-title: ...
          exclude-workflow-initiator-as-approver: false
```  

The 1.8.0 version is in use.
Once the issue is raised, and a comment like "approve" is placed, nothing is happening.
There is no another action from bot, that would close the issue and move forward the pipeline, it will stay in waiting for 6hrs.

What is wrong? Is it a bug or am I missing something?

**Comment by Woitekku** [comment]
oh i guess i found the issue 
`exclude-workflow-initiator-as-approver: false`

**Comment by Woitekku** [comment]
no; it should be false, as per default to be able to do self-approval...

**Comment by trstringer** [comment]
Can you confirm what the list of approvers are on the actual issue itself?

**Comment by snskArora** [comment]
Hi @Woitekku 

Do you have any inputs for us, The additional Information requested by thomas will help us in debuging this issue.
